### PR TITLE
feat: Create attendee pdf for every ticket holder

### LINKED ISSUE #7397: Ticket PDF should exist for each attendee
Ticket PDF is currently associated with an order which should instead be associated with an attendee as each attendee should have a ticket

### PR DESCRIPTION
**Pull request by codedsun**
Fixes #7397 

#### Short description of what this resolves:
feat: Create attendee pdf for every ticket holder

#### Changes proposed in this pull request:

-
-
-

#### Checklist

- [ ] I have read the [Contribution & Best practices Guide](https://blog.fossasia.org/open-source-developer-guide-and-best-practices-at-fossasia) and my PR follows them.
- [ ] My branch is up-to-date with the Upstream `development` branch.
- [ ] The unit tests pass locally with my changes 
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] I have added necessary documentation (if appropriate)

- [ ] All the functions created/modified in this PR contain relevant docstrings.

### CODE REVIEW

**Comment by codedsun** [comment]
@iamareebjamal - Review now

Behaviour 
image


Tested the frontend also, working fine

File: app/api/helpers/order.py [hunk 1]
```diff
@@ -88,41 +88,35 @@ def create_pdf_tickets_for_holder(order):
...
-                order_tickets=order_tickets,
-            ),
-            UPLOAD_PATHS['pdf']['order'],
-            dir_path='/static/uploads/pdf/tickets/',
-            identifier=order.identifier,
-            upload_dir='generated/invoices/',
-        )
-        save_to_db(order)
+    # create order invoices pdf
+    order_tickets = OrderTicket.query.filter_by(order_id=order.id).all()
```
Comment (iamareebjamal) [inline app/api/helpers/order.py:104]: I believe this indentation is wrong

Comment (codedsun) [reply app/api/helpers/order.py]: Fixed!

**Comment by codedsun** [comment]
@iamareebjamal - Review now!

**Review by iamareebjamal — APPROVED** [review]

**Pull request merged**